environments/liveness/liveness-environment/amplify/backend/function/livenessenvironment6105cfac/src/index.js

```diff
@@ -0,0 +1,33 @@
+import { RekognitionLiveness } from '@amzn/aws-rekognition-liveness-js-v2-client';
+
+/**
+ * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
+ */
+
+export const handler = async (event, req) => {
+  const client = new RekognitionLiveness({
+    region: 'us-east-1',
+    endpoint: `https://us-east-1.gamma.frontend.reventlov.rekognition.aws.dev`,
+  });
+
+  const response = await client
+    .createFaceLivenessSession({
+      AwsAccountId: '845721723350',
+      Settings: {
+        ChallengePreferences: [
+          { Type: 'FaceMovementChallenge' },
+          { Type: 'FaceMovementAndLightChallenge' },
```

Comment:
Is it possible to pass in the challenge list we want to use at runtime? that way on our sample app we can choose which challenges to run?

Reply:
Updated to require a selection before the check screen that gets passed to the lambda, lmk what you think!